Client/Frontend/Browser/ContextMenuHelper.swift

```diff
@@ -24,14 +24,17 @@ class ContextMenuHelper: NSObject {
 
     required init(tab: Tab) {
         super.init()
-
         self.tab = tab
+    }
 
-        nativeHighlightLongPressRecognizer = gestureRecognizerWithDescriptionFragment("action=_highlightLongPressRecognized:") as? UILongPressGestureRecognizer
+    func setup() {
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.2) {
+            self.nativeHighlightLongPressRecognizer = self.gestureRecognizerWithDescriptionFragment("action=_highlightLongPressRecognized:") as? UILongPressGestureRecognizer
 
-        if let nativeLongPressRecognizer = gestureRecognizerWithDescriptionFragment("action=_longPressRecognized:") as? UILongPressGestureRecognizer {
-            nativeLongPressRecognizer.removeTarget(nil, action: nil)
-            nativeLongPressRecognizer.addTarget(self, action: #selector(longPressGestureDetected))
+            if let nativeLongPressRecognizer = self.gestureRecognizerWithDescriptionFragment("action=_longPressRecognized:") as? UILongPressGestureRecognizer {
+                nativeLongPressRecognizer.removeTarget(nil, action: nil)
+                nativeLongPressRecognizer.addTarget(self, action: #selector(self.longPressGestureDetected))
+            }
```

Comment:
Consider instead only calling the `asyncAfter` hook if the `if let nativeLongPressRecognizer =` fails. Although I am not convinced that is any more reliable that this method.